index.html

```diff
@@ -255,6 +255,62 @@
 
 ---
 
+# Continuous Analysis
+
+The Status Quo Doesn't Work:
+.task[
+- 66% of microarray gene expression experiments could not be reproduced without discrepancy (Ioannidis et al. 2009)
+
+- 50% of microarray gene experssion experiments can not be reproduced because of version number alone (Beaulieu-Jones & Greene 2016)
+
+]
+
+Even in the good cases:
+
+.task[
+- 280 hours for non-expert to reproduce drug-target network (Garijo et al. 2013) 
+]
+
+---
+
+# Continuous Analysis
+
+.task[
+- Docker - container technology similar to virtual machines, allows you to specify and share exact computing environment
+
+- Continuous Integration - software development technique used to test and deploy source code to production servers.
+]
+.center[
+<img src="images/continuous_analysis.png" alt="Drawing" style="height: 340px;"/>
+]
+
+---
+
+## Output 1: Containers match original computing environment.
+
+.center[
+<img src="images/containers.png" alt="Drawing" style="height: 380px;"/>
+]
+
+
+---
+
+## Output 2: Figures are synchronized to code and changes are easy to visualize.
+
+.center[
+<img src="images/figures.png" alt="Drawing" style="height: 380px;"/>
+]
+
+---
+
+## Output 3: Audit logs show exactly what analyses were peformed (great for private data).
+
+.center[
+<img src="images/logs.png" alt="Drawing" style="height: 380px;"/>
+]
+
+---
+
 # What tools exist to help out?
```

Comment:
What about making a GitHub issue where you post about CA. You could then edit by linking from here directly to your example. Just a thought of a different way that you might introduce CA into this discussion.